addons/web/static/src/js/views/kanban/kanban_controller.js

```diff
@@ -302,6 +305,22 @@ var KanbanController = BasicController.extend({
                 this.$('.o_kanban_record:first').focus();
         }
     },
+    /**
+     * Bounce the 'Create' button.
+     *
+     * @private
+     * @param {OdooEvent} ev
+     **/
+    _onClick: function (ev) {
+        var state = this.model.get(this.handle, {raw: true});
+        if (!state.count) {
```

Comment:
Suggestion: move the check on this.buttons here
`if (!state.count && this.buttons)`